src/Compilers/Core/Portable/CommandLine/SarifDiagnosticComparer.cs

```diff
@@ -47,7 +47,7 @@ public bool Equals(DiagnosticDescriptor x, DiagnosticDescriptor y)
             }
 
             // The properties are guaranteed to be non-null by DiagnosticDescriptor invariants.
-            Debug.Assert(x.Description != null && x.Title != null && x.CustomTags != null);
+            Debug.Assert(x is { Description: { }, Title: { }, CustomTags: { } });
             Debug.Assert(y.Description != null && y.Title != null && y.CustomTags != null);
```

Comment:
Why didn't this line get the treatment?

Reply:
The script I ran stops after applying one change within a given document. Helps us see each single application of the refactoring in context.